### PR TITLE
Adding a favicon to satis webpage

### PR DESCRIPTION
**Pull request by moneef**
Refs #194 
Icon source: http://commons.wikimedia.org/wiki/File:Package-x-generic.svg

### CODE REVIEW

File: views/index.html.twig [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="robots" content="noindex,nofollow" />
+    <link rel="shortcut icon" type="image/png" href="/satis/views/assets/img/favicon.png">
```
Comment (GromNaN) [inline views/index.html.twig:9]: This absolute path is specific to your deployment. The file must be duplicated into the "output" directory and the path must be relative.

Comment (alcohol) [reply views/index.html.twig:9]: Yeah @GromNaN is correct. Satis is basically a static site generator. The output directory is configurable, but any assets should be placed there along with the generated html. Alternatively, you could inline the png like we did with most other javascript/css/image assets. See https://stackoverflow.com/a/5568484 for an example.

**Comment by glensc** [comment]
For future generations add origin of the icon source (describe in PR body). So it could be updated or possible license issues can be addressed.

**Comment by alcohol** [comment]
The source is referenced in the issue. True, it's indirect, but it is known.

**Comment by glensc** [comment]
There is no issue reference, the PR body is completely empty.

**Comment by alcohol** [comment]
As pedantic as ever. Anyway, I see the link clear as day on my screen. But there you go, edited the PR body so you can manage to find it too.

**Pull request closed without merging**